Core/Inc/FreeRTOSTasks/TCHandlingTask.hpp

```diff
@@ -1,34 +1,72 @@
 #pragma once
-
 #include "Task.hpp"
 #include "COBS.hpp"
+#include "queue.h"
+/*
+The HAL_UARTEx_ReceiveToIdle_DMA() function allows to handle reception of Data from Hyperterminal
+using DMA and notify application of already received data while the reception is still ongoing.
+Received characters are handled by DMA and are stored in the user aRXBufferUser buffer.
+Notifications to application, that some data are available in reception buffer, are done
+through the execution of a user callback : HAL_UARTEx_RxEventCallback().
+This callback will be executed when any of following events occurs :
+- HT (Half Transfer) : Half of Rx buffer is filled)
+- TC (Transfer Complete) : Rx buffer is full.
+(In case of Circular DMA, reception could go on, and next reception data will be stored
+    in index 0 of reception buffer by DMA).
+- Idle Event on Rx line : Triggered when RX line has been in idle state (normally high state)
+for 1 frame time, after last received byte.
+*/
+
+#define queue_length 10
```

Comment:
Move to `COMMS_Definitions`

Reply:
Έκανα push again και έσβησα οτι είχε να κάνει με queues γιατί τώρα πια δε χρειάζονται